text/0000-deprecate-computed-clobberability.md

```diff
@@ -123,6 +123,18 @@ functions elsewhere and replace them with a proxy getter and the mandatory
 setter assertion, respectively. In the long term, making computeds more
 transparent in this way would be ideal, but it is out of scope for this RFC.
 
+## Deprecation Timeline
+
+This change will be a breaking change, which means we will not be able to change
+the behavior of `computed` until Ember v4.0.0. Until then, we should provide two
+deprecation warnings:
+
+1. When users invoke the default override-setter behavior
+2. When users call `.readOnly()` on a computed property
```

Comment:
If we deprecate this, users have no way of getting a hard error (not a deprecation warning) when they clobber without getting deprecation warnings by using readOnly.

I recommend only deprecating .readOnly() in the 4.x era. Yeah, like I said, this is a pain.